pkg/i18n/english.go

```diff
@@ -1970,6 +1975,12 @@ func EnglishTranslationSet() *TranslationSet {
 		CustomCommands:                           "Custom commands",
 		NoApplicableCommandsInThisContext:        "(No applicable commands in this context)",
 		SelectCommitsOfCurrentBranch:             "Select commits of current branch",
+		ViewMergeConflictOptions:                 "View merge conflict options",
+		ViewMergeConflictOptionsTooltip:          "View options for resolving merge conflicts.",
+		MergeConflictOptionsTitle:                "Resolve merge conflicts",
+		UseHead:                                  "Use HEAD",
```

Comment:
For non-textual conflicts we settled on the terms "current changes" (for head) and "incoming changes"; see #4431, so I'd suggest to use those here, too.

Also, we don't use Title Case in menu items, so it should be "Use current changes" and "Use both".